plugins/woocommerce/src/StoreApi/Utilities/CartController.php

```diff
@@ -112,9 +112,14 @@ public function add_to_cart( $request ) {
 			$request['cart_item_data']
 		);
 
-		$this->validate_add_to_cart( $product, $request );
+		$quantity_limits = new QuantityLimits();
+
+		// If quantity was not passed, it should default to the minimum allowed quantity.
```

Comment:
The `wp_parse_args` above would not add `quantity => 1` _unless_ quantity had been removed by the `woocommerce_store_api_add_to_cart_data` filter